### PR TITLE
Information about loading data into the Bridgehead's FHIR store

### PR DESCRIPTION
**Pull request by DavidCroftDKFZ**
Added this information because many sites have asked about it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -311,6 +312,29 @@ Once you edited the gitlab config, the bridgehead will autoupdate the config wit
...
 There will be a delay before the effects of Directory sync become visible. First, you will need to wait until the time you have specified in ```TIMER_CRON```. Second, the information will then be synchronized from your national node with the central European Directory. This can take up to 24 hours.
 
+### Loading data
+
+The data accessed by the federated search is held in the Bridgehead in a FHIR store (we use Blaze).
+
+You can load data into this store by using its FHIR API:
+
+```
+https://<Name of your server>/bbmri-localdatamanagement/fhir
```
Comment (patrickskowronekdkfz) [inline README.md:322]: Please replace the BBMRI with project.

File: README.md [hunk 2]
```diff
@@ -311,6 +312,29 @@ Once you edited the gitlab config, the bridgehead will autoupdate the config wit
...
+```
+The name of your server will generally be the full name of the VM that the Bridgehead runs on. You can alternatively supply an IP address.
+
+The FHIR API uses basic auth. You can find the credentials in `/etc/bridgehead/bbmri.local.conf`.
```
Comment (patrickskowronekdkfz) [inline README.md:326]: Please replace BBMRI with the project.

File: README.md [hunk 3]
```diff
@@ -311,6 +312,29 @@ Once you edited the gitlab config, the bridgehead will autoupdate the config wit
...
+
+Note that if you don't have a DNS certificate for the Bridgehead, you will need to allow an insecure connection. E.g. with curl, use the `-k` flag.
+
+#### ETL for BBMRI and GBA
+
+Normally, you will need to build your own ETL to feed the Bridgehead. However, there are two cases where a short cut might be available:
+- If you are a German biobank and you already feed your data into a Data Integration Center (DIZ). In this case, you can use the [TransFAIR tool](https://github.com/samply/transFAIR) with the `bbmri2mii` profile to pull data from the DIZ and load it into the Bridgehead's FHIR store. You will need to discuss this with your local DIZ management before proceeding with this approach.
```
Comment (patrickskowronekdkfz) [inline README.md:333]: Currently still in development, I would add it, when it is ready.

**Review by patrickskowronekdkfz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DavidCroftDKFZ) [reply README.md:322]: Done

**Review by patrickskowronekdkfz — APPROVED** [review]

**Pull request merged**